driver-core/src/main/java/com/datastax/driver/core/KeyspaceMetadata.java

```diff
@@ -262,21 +286,61 @@ public String exportAsString() {
 
         sb.append(asCQLQuery()).append('\n');
 
-        for (UserType udt : userTypes.values())
+        // include types, tables, views, functions and aggregates, each ordered by name, with one small exception
+        // being that user types are ordered topologically and then by name within same level.
+        for (UserType udt : getSortedUserTypes())
```

Comment:
I was unsure if maintaining the topographical order in `KeyspaceMetadata.userTypes` was important or not.  If it's not it would simplify some code / remove some semi-duplication, if agree I'll remove it.